tests/ci/run_s2n_interop_test.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+set -ex
+
+source tests/ci/common_posix_setup.sh
+
+scratch_folder=${SYS_ROOT}/"s2n-scratch"
+s2n_url='https://github.com/aws/s2n-tls.git'
+s2n_branch='main'
+lc_url='https://github.com/aws/aws-lc.git'
+lc_branch='main'
+
+mkdir -p "${scratch_folder}"
+rm -rf "${scratch_folder:?}"/*
+
+# clone s2n-tls
+git clone --depth 1 --branch "${s2n_branch}" "${s2n_url}" "${scratch_folder}/s2n-tls"
+
+# clone aws-lc
+git clone --depth 1 --branch "${lc_branch}" "${lc_url}" "${scratch_folder}/s2n-tls/aws-lc"
```

Comment:
This shouldn't need to clone AWS-LC since it will already be there when run locally or run in codebuild.